index.js

```diff
@@ -9,8 +9,16 @@ const validate = require('validate-npm-package-name');
 
 class InvalidNameError extends Error {}
 
+const orgRegex = /^@[a-z\d][\w-.]+\/?$/;
+
 const request = async (name, options) => {
 	const registryUrl = normalizeUrl(options.registryUrl || configuredRegistryUrl);
+	const npmOrgUrl = 'https://www.npmjs.com/org/';
+
+	const isOrg = orgRegex.test(name);
```

Comment:
You can just do `name.startsWith('@')`.

And:
```suggestion
	const isOrganization = orgRegex.test(name);
```

Reply:
`name.startsWith('@')` would be true even for scoped packages like `@ava/init`